packages/components/src/tools-panel/tools-panel-header/component.js

```diff
@@ -108,14 +109,23 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 
 	const defaultItems = Object.entries( menuItems?.default || {} );
 	const optionalItems = Object.entries( menuItems?.optional || {} );
+	const dropDownMenuIcon = areOptionalControlsAvailableAndHidden
+		? plus
+		: moreVertical;
+	const dropDownMenuLabelText = areOptionalControlsAvailableAndHidden
+		? _x(
+				'Show and add options',
+				'Button label to reveal tool panel options'
+		  )
+		: _x( 'Show options', 'Button label to reveal tool panel options' );
```

Comment:
Since the menu isn't only used to "show" and "add" option, but also to "hide", "remove" and "reset" them, maybe we should use a term that is different from all the ones I've just listed. I think your initial suggestion of "View" may be a bit safer — and I also thought about "Manage" as an alternative.

Another quick thought — what if, instead of "options", we used  "sections"?

To sum it up, a few alternatives could be:

- "Manage sections" / "Manage and add sections"
- "Manage options" / "Manage and add options"
- "View sections" / "View and add sections"
- "View options" / "View and add options"

Reply:
Thanks for thinking about this. I'll flip the verb back to "View" for now since I agree that it communicates more than "Show".

I'm fairly neutral on "sections". If folks think it describes the controls more succinctly/clearly I'm very willing to defer to those opinions. 👍